stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.scss

```diff
@@ -69,10 +70,13 @@
         line-height: 0;
         list-style-type: none;
         margin-bottom: $spacing-24;
+        text-transform: uppercase;
       }
     }
   }
-
+.mega-title{
```

Comment:
@Ashish-srij Kindly fix alignment and also format it properly.

Reply:
@shishirphadke Done Please check.